packages/react/src/tags/docs/overview.mdx

```diff
@@ -59,9 +59,13 @@ A list of related tags without links.
 
 ## Removable tags
 
-Tags can be removed from a list, if you add an `onRemove` function.
+Tags can be removed from a list when you pass a function to the `onRemove` prop.
 
-Please avoid using Tags with links and remove buttons together, as it is confusing for users.
+When a user removes a tag, their focus is automatically moved to the previous tag in the list. When the last tag is removed, focus is not moved, and, depending on the context, may need to be manually moved in `onRemove` to help the user continue their task.
+
+Screen reader users need to be informed when a tag has been removed. Depending on the context, this may be done explicitly, like in the example below, or more commonly implicitly by announcing the resultant changes to the page (like in the [Search filter pattern](/patterns/search-filters)).
```

Comment:
I think this would read better as three sentences, i.e. full stop after `below`. Also, the double adverb is difficult to say. Do you need `more commonly`? If so, what do you think of `The most common approach is implicitly announcing...`.

Reply:
Yeah, I wasn't too keen on my wording either

Reply:
I've just pushed some more wordsmithing